src/service/Review.ts

```diff
@@ -23,6 +32,72 @@ export interface ReviewActionResult {
     };
 }
 
+export async function findCodeReviewRecord ({id, pullRequestLink}: {id?: number; pullRequestLink?: string}): Promise<CodeReviewRecord | null> {
+    let where;
```

Comment:
Shouldn't this need a type?

Reply:
Only if we want to.  Because of prisma client auto inherit the `CodeReviewWhereInput` type.  If you make a mistake, error will be throw on line 39 & 41 if those fields don't match the model.
eg. line 39: `where = {id, test: 'tst'};` will throw error on line 39 like this.  eslint won't pick this up, but phpstorm do pick up this error.  Pretty nice.

```
S2322: Type '{ id: number; test: string; } | { pullRequestLink: string; }' is not assignable to type 'CodeReviewWhereInput | undefined'.   Type '{ id: number; test: string; }' is not assignable to type 'CodeReviewWhereInput'.     Object literal may only specify known properties, and 'test' does not exist in type 'CodeReviewWhereInput'.
```

Pretty interesting how prisma enforce the typing.

We can declare the var like this to be explicit: `where: Prisma.CodeReviewWhereInput | null = null` but not required.